src/java/Logica/Entidades/Valoracion.java

```diff
@@ -0,0 +1,76 @@
+package Logica.Entidades;
+
+import java.io.Serializable;
+import javax.persistence.Basic;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Valoracion implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Integer id;
+
+    @Basic
+    private Integer idPaper; // ID del paper asociado
+    @Basic
+    private Integer idUsuario; // ID del usuario que hace la valoración
```

Comment:
@patricioperrone te menciono lo mismo que le dije a Bruno en su PR. Acá en vez de idUsuario deberias poner Lector utilizando la anotacion @OneToOne de JPA asi despues podemos acceder a los metodos del Lector desde ahi. Revisa esta docu https://www.geeksforgeeks.org/jpa-one-to-one-mapping/